src/mmw/js/src/modeling/models.js

```diff
@@ -1003,8 +1003,20 @@ var ScenariosCollection = Backbone.Collection.extend({
             newModel = new ScenarioModel({
                 is_current_conditions: false,
                 name: this.makeNewScenarioName('Copy of ' + source.get('name')),
+                user_id: source.get('user_id'),
                 inputs: source.get('inputs').toJSON(),
-                modifications: source.get('modifications').toJSON()
+                inputmod_hash: source.get('inputmod_hash'),
+                modifications: source.get('modifications').toJSON(),
+                modification_hash: source.get('modification_hash'),
+                active: source.get('active'),
+                job_id: source.get('job_id'),
+                poll_error: source.get('poll_error'),
+                results: source.get('results'),
+                has_fetched_results: source.get('has_fetched_results'),
+                aoi_census: source.get('aoi_census'),
+                modification_censuses: source.get('modification_censuses'),
+                allow_save: source.get('allow_save'),
+                options_menu_is_open: source.get('options_menu_is_open'),
```

Comment:
`options_menu_is_open` and `active` probably shouldn't be copied over; they're used by the view.